float64 average_velocity
